sandman_web/sandman_web/reports/templates/report.html

```diff
@@ -1,6 +1,7 @@
 {% extends 'base.html' %}
 
 {% block stylesheet %}
+  <link rel="stylesheet" href="{{ url_for('static', filename='style.css') }}">
```

Comment:
I think this line isn't needed because the report stylesheet imports the main stylesheet already. Did you notice issues that made you add this?

Reply:
This was a fix as the import wasn't working but I hadn't realized that was the real issue. I will remove this line as it is unnecessary after fixing the import url.